lib/libzfs_core/libzfs_core.c

```diff
@@ -169,6 +171,29 @@ lzc_ioctl(zfs_ioc_t ioc, const char *name,
 	return (error);
 }
 
+static int
+lzc_ioctl(const char *cmd, const char *name, nvlist_t *source,
```

Comment:
I asked during an earlier review; the answer was this was to let "options remain separate from actual data (e.g. a list of properties to set) so that we don't get pesky namespace collisions or inconsistencies from future extensions"

It also exists as an nvlist rather than a bitmask for said extensions.